sw/device/silicon_creator/lib/drivers/pinmux.c

```diff
@@ -0,0 +1,69 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/silicon_creator/lib/drivers/pinmux.h"
+
+#include "sw/device/lib/base/memory.h"
+#include "sw/device/silicon_creator/lib/base/abs_mmio.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+#include "pinmux_regs.h"  // Generated.
+
+/**
+ * A peripheral input and MIO pad to link it to.
+ */
+typedef struct pinmux_input {
+  top_earlgrey_pinmux_peripheral_in_t periph;
+  top_earlgrey_pinmux_insel_t pad;
+} pinmux_input_t;
+
+/**
+ * Peripheral input to MIO pad mappings.
+ */
+static const pinmux_input_t pinmux_inputs[] = {
```

Comment:
`kPinmuxInputs`, since it's a constant. (Applies to outputs too.)

Reply:
One day I'll send a PR where I haven't forgotten to do this somewhere... Thanks, done.

Reply:
It's a weird convention we inherited from Google, so I don't blame you. =)